CHANGELOG.md

```
@@ -32,6 +32,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
 
 - Hotfix to ensure compatibility with PyTorch v2.6 when loading weights files.
+- Upgraded minimum Lightning version to 2.6
+- Changed default value for gradient_clip_val to be 1.0 
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)